cumulusci/cli/cci.py

```diff
@@ -1024,7 +1025,11 @@ def task_doc(config):
 @click.argument("task_name")
 @pass_config(load_keychain=False)
 def task_info(config, task_name):
-    task_config = config.project_config.get_task(task_name)
+    try:
+        task_config = config.project_config.get_task(task_name)
+    except (CumulusCIUsageError, TaskNotFoundError) as e:
+        raise click.UsageError(str(e))
```

Comment:
TaskNotFoundError is a subclass of CumulusCIUsageError; you don't have to catch it explicitly.

Reply:
Oh and this branch needs a test for coverage.